dbtype/mssql/deployed_check.sh

```diff
@@ -5,8 +5,8 @@ function deployed_check() {
   if [ $? -eq 0 ]
   then
 
-    if [ `${db_binary} -d ${deployment_db} ${server_flag}${port_flag} ${user_flag} ${password_flag} -h -1 -b -Q "SET NOCOUNT ON; select coalesce ((select count(*) from deployment_tracker t
-    where dbname = '${dbname}'
+    if [ `${db_binary} -d ${deployment_db} -n ${db_destination_name} ${server_flag}${port_flag} ${user_flag} ${password_flag} -h -1 -b -Q "SET NOCOUNT ON; select coalesce ((select count(*) from deployment_tracker t
```

Comment:
What are you trying to do with -n here?  sqlcmd is telling me its an obsolete option.